src/cryptonote_core/service_node_rules.h

```diff
@@ -5,8 +5,19 @@
 #include "service_node_voting.h"
 
 namespace service_nodes {
-  constexpr size_t PULSE_QUORUM_ENTROPY_LAG = 21; // How many blocks back from the tip of the Blockchain to source entropy for the Pulse quorums.
-  constexpr size_t PULSE_QUORUM_SIZE        = 11;
+  constexpr size_t PULSE_QUORUM_ENTROPY_LAG             = 21; // How many blocks back from the tip of the Blockchain to source entropy for the Pulse quorums.
+  constexpr size_t PULSE_QUORUM_NUM_VALIDATORS          = 11;
+  constexpr size_t PULSE_QUORUM_SIZE                    = PULSE_QUORUM_NUM_VALIDATORS + 1 /*Leader*/;
+
+  // NOTE: PULSE_QUORUM_SIZE + 1 for when round > 0. We remove the
+  // block_winner that failed in round 0 from the active list and we need
+  // 1 new leader from the list to take its spot.
+  // i.e. Service Node List must have 11 Validators + 1 Original Leader + 1 Optional New Leader
+  constexpr size_t PULSE_MIN_SERVICE_NODES              = PULSE_QUORUM_SIZE + 1 /*Optional for Leader when (Round > 0)*/;
```

Comment:
> This is the gating between falling back to PoW and Pulse so I feel like being careful with our numbers here is warranted.

Right, that's why I thought it might be something configurable separately.  If you only have 11 nodes, turning on PoS with those 11 nodes sounds a bit scary.  (Not that 20 sounds all that much better).

On the other hand, for testnet it'd be nice to have this as low as is feasible.

Reply:
Any less than 50 nodes we should probably fall back to PoW, PoW should be more reliable in that setting.

Reply:
Done. We might want a % situation here like we do with uptime testing but it's hard to say until we see it in practice.